src/test/java/hudson/plugins/testlink/result/ResultSeekerTest.java

```diff
@@ -0,0 +1,33 @@
+package hudson.plugins.testlink.result;
+
+import net.sf.json.JSONObject;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.jvnet.hudson.test.For;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.kohsuke.stapler.RequestImpl;
+import org.mockito.Mock;
+
+import javax.management.Descriptor;
+
+@For(ResultSeeker.class)
+@Ignore
```

Comment:
specify a reason in the annotation value

Reply:
Oops, I pushed an unfinished test.
It does not hurt in the current state, but I will remove the file